### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by elliotBraem**
In attempting to run and develop the contract locally, I've been writing some documentation to help others that plan to do the same.

Please leave any comments for how to do anything better/if I do something incorrectly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+
+1. [NEAR CLI RS](https://github.com/near/near-cli-rs), to deploy and interact with the contract.
+2. [cargo-near](https://github.com/near/cargo-near), to easily create testnet accounts.
+
+## Building
+
+From the root directory, run:
+
+```cmd
+cargo build
```
Comment (ailisp) [inline README.md]: we use `./build.sh` to specify some flags to build

Comment (ailisp) [reply README.md]: Now we also need to build community and community factory contract:
```
cd community
./build.sh
cd ../community-factory
./build.sh
cd ..
./build.sh
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+```cmd
+cargo test
+```
+
+## Deploying
+
+Using [NEAR CLI RS](https://github.com/near/near-cli-rs), run the following command. Be sure to set your own account id and corresponding network.
+
+```cmd
+near contract deploy {{account.near}} use-file ./target/wasm32-unknown-unknown/release/devgovgigs.wasm without-init-call network-config {{env}}
```
Comment (ailisp) [inline README.md:42]: and deploy community factory
```
near contract deploy {{community.account.near}} use-file ./target/wasm32-unknown-unknown/release/devhub_community_factory.wasm without-init-call network-config {{env}}

File: README.md (hunk 1 above)
Comment (frol) [inline README.md]:
```suggestion
cd community
./build.sh
cd ../community-factory
./build.sh
cd ..
./build.sh
```

**Review by frol — APPROVED** [review]

**Pull request merged**